go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/0xPolygon/polygon-edge
 
-go 1.18
+go 1.19
```

Comment:
Can we bump up the minimum required Go version into `1.19`? This would simplify the use of atomics with `atomic.Bool`. If this is not feasible, I'll change the atomics to use the older API.

Reply:
@tuommaki We'll need to conduct a series of tests to upgrade the Go version, that will take some time, if you need this merged ASAP consider to make it 1.18 compatible.

Reply:
I changed the code to use older `atomic` functions and `*uint64` . The CI is somehow requesting for a token now though :thinking: